pymeasure/instruments/hp/hp11713a.py

```diff
@@ -0,0 +1,173 @@
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+import logging
+from pymeasure.instruments import Instrument, Channel
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+Attenuator_11dB = {
+    0: (False, False, False, False),
+    1: (True, False, False, False),
+    2: (False, True, False, False),
+    3: (True, True, False, False),
+    4: (False, False, False, True),
+    5: (True, False, False, True),
+    6: (False, True, False, True),
+    7: (True, True, False, True),
+    8: (False, False, True, True),
+    9: (True, False, True, True),
+    10: (False, True, True, True),
+    11: (True, True, True, True),
+}
+""" Mapping of logical values for use with 0 - 11 dB attenuators """
+
+Attenuator_110dB = {
+    0: (False, False, False, False),
+    10: (True, False, False, False),
+    20: (False, True, False, False),
+    30: (True, True, False, False),
+    40: (False, False, False, True),
+    50: (True, False, False, True),
+    60: (False, True, False, True),
+    70: (True, True, False, True),
+    80: (False, False, True, True),
+    90: (True, False, True, True),
+    100: (False, True, True, True),
+    110: (True, True, True, True),
+}
+""" Mapping of logical values for use with 0 - 110 dB attenuators """
+
+Attenuator_70dB_3_Section = {
+    0: (False, False, False, False),
+    10: (True, False, False, False),
+    20: (False, True, False, False),
+    30: (True, True, False, False),
+    40: (False, False, True, False),
+    50: (True, False, True, False),
+    60: (False, True, True, False),
+    70: (True, True, True, False),
+}
+""" Mapping of logical values for use with 0 - 70 dB attenuators with 3 switching sections """
+
+Attenuator_70dB_4_Section = {
+    0: (False, False, False, False),
+    10: (True, False, False, False),
+    20: (False, False, False, True),
+    30: (True, False, False, True),
+    40: (False, True, False, True),
+    50: (True, True, False, True),
+    60: (False, True, True, True),
+    70: (True, True, True, True),
+}
+""" Mapping of logical values for use with 0 - 70 dB attenuators with 4 switching sections """
+
+
+class SwitchDriverChannel(Channel):
+    enabled = Instrument.setting(
+        "%s{ch}",
+        """
+        Set this channel to the polarity 'A' for True and 'B' for False.
+        """,
+        map_values=True,
+        values={True: "A", False: "B"}
+    )
+
+
+class HP11713A(Instrument):
+    """
+    Represents the HP 11713A Switch and Attenuator Driver and provides a high-level
+    interface for interacting with the instrument.
+
```

Comment:
Do you consider it opportun to describe the attenuator definition via dictionaries, such that users understand, that they have to do it?